app/src/main/res/values-uk/strings_pin.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="pin">Пароль</string>
```

Comment:
can you please check if we really need to have this string (for `en` too )? It seems to me that we use it somewhere and then it's overriden by something.

Reply:
We used this string in mobile_navigation in label 
![image](https://user-images.githubusercontent.com/109308856/189682186-8b2f76f5-b90a-4ad1-8fc1-111375b2a6b5.png)

Reply:
right, but is the label visible somewhere?

Reply:
i guess this label is not visible, because we haven't ActionBar in pin fragment, right?